docs/public/debugging-using-devfile.adoc

```diff
@@ -1,29 +1,11 @@
-# Debugging components using devfiles
-
-`odo` uses devfiles to build and deploy components. More information on devifles : https://redhat-developer.github.io/devfile/[Introduction to devfile]
-
-In order to use `odo debug` your devfile is required to have a `debugrun` step. Example of a nodejs devfile with a debugrun step:
-
-```yaml
-schemaVersion: 2.0.0
-metadata:
-  name: nodejs
-  version: 1.0.0
-starterProjects:
-  - name: nodejs-starter
-    git:
-      remotes:
-        origin: "https://github.com/odo-devfiles/nodejs-ex.git"
-components:
-  - name: runtime
-    container:
-      image: registry.access.redhat.com/ubi8/nodejs-12:1-45
-      memoryLimit: 1024Mi
-      mountSources: true
-      sourceMapping: /project
-      endpoints:
-        - name: http-3000
-          targetPort: 3000
+= Debugging a component
+
+Debugging your component involves port forwarding with the Kubernetes pod. Before you start, it is required that you have a `debugrun` step located within your `devfile.yaml`.
```

Comment:
Wondering why you use `debugrun`  here instead of `debug`  ?   You need a `kind: debug` command (or did this change?).

Reply:
Yes this should be any step with `kind: debug`